platform/security/security-util/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>ddf.platform.security</groupId>
+        <artifactId>security</artifactId>
+        <version>2.12.0-SNAPSHOT</version>
+    </parent>
+    <groupId>ddf.security</groupId>
+    <artifactId>security-util</artifactId>
```

Comment:
Security-util is not descriptive of what this bundle is for this could end up looking like another ddf-security-common mess.
We have so many utility classes in ddf-security-common that are related SAML it breaks my heart to add another bundle and not pull them into this one or put this into there since they have a common SAML dependency(would prefer the former but it is probably a lot of work)

Reply:
I started to move this stuff to `ddf-security-common`, but it looks like that's a `jar` vs my `bundle`. I think it's better to use bundles than jars, so I think I should avoid refactoring this code to `ddf-security-common`.

Reply:
This does seem like it belongs in security-common to me. That jar probably needs a little cleanup anyways, so it is something that can wait. Also to follow our naming guidelines, it would need to be like `security-saml-util` or something similar.

Reply:
fixed